### DISCUSSION TITLE
Hello I accidentally blocked my ip address

### DISCUSSION THREAD

**Discussion by Batyray** [NginxProxyManager/nginx-proxy-manager#4180] [Q&A]
Hello I accidentally blocked my IP address and can not get nginx manager address how can I change access list from backend does anybody knows?

**Comment by shalafi99** [comment]
Hello, I think the access lists are configured under the `location /` block in each conf file inside the NPM container's data volume.

An example, for this proxy host below:
![image](https://github.com/user-attachments/assets/3bee963e-79cc-4eed-98ee-10ee18169a03)

The conf file would be /data/nginx/proxy_host/4.conf
